tests/osaka/eip7883_modexp_gas_increase/test_eip_mainnet.py

```diff
@@ -0,0 +1,63 @@
+"""
+EIP-7883 ModExp gas cost increase tests.
+
+Tests for ModExp gas cost increase in
+[EIP-7883: ModExp Gas Cost Increase](https://eips.ethereum.org/EIPS/eip-7883).
```

Comment:
Could we make it clear in the docstring that these are specifically for mainnet execute use only (fill is a sanity check).
```suggestion
Mainnet Marked Checklist Tests for ModExp gas cost increase in
[EIP-7883: ModExp Gas Cost Increase](https://eips.ethereum.org/EIPS/eip-7883).
```